NOTICE

```diff
@@ -239,6 +239,35 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
+----------------------------------------------------------------------------
+tinycbor 
+
+MIT License
+
+Copyright (c) 2017 Intel Corporation
+

```

Comment:
Shouldn't it be 2021 and not 2017?

Reply:
That is for tinycbor license, copied from its website.